cmd/neofs-lens/internal/meta/root.go

```diff
@@ -0,0 +1,32 @@
+package meta
+
+import (
+	"math"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	vAddress string
+	vPath    string
+)
+
+type epochState struct{}
+
+func (s epochState) CurrentEpoch() uint64 {
+	return math.MaxUint64
```

Comment:
Looks like a fake, what is this workaround for?

Reply:
after #1634 the meta panics if no epoch state is presented. so that is just for returning all the objects in it

we can add a flag that specifies the current epoch but not sure if it is useful